src/main/java/org/cactoos/collection/CollectionEnvelope.java

```diff
@@ -137,4 +137,11 @@ public final void clear() {
             "#clear(): the collection is ready only"
         );
     }
+
+    // @checkstyle DesignForExtensionCheck (5 lines)
```

Comment:
@driver733 Why is this needed here?

Reply:
@amihaiemil Because according to checkstyle,
The overridden methods should not have javadocs. 
And
>  If the method which can be overridden in a subclass has a javadoc comment (a good practice is to explain its self-use of overridable methods) the check will not rise a violation. The violation can also be skipped if the method which can be overridden in a subclass has one or more annotations that are specified in ignoredAnnotations option. Note, that by default @Override annotation is not included in the ignoredAnnotations set as in a subclass the method which has the annotation can also be overridden in its subclass.